test/src/test/java/org/axonframework/test/aggregate/StubAggregateLifecycleExtensionTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2010-2018. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.test.aggregate;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.*;
+
+import static org.axonframework.modelling.command.AggregateLifecycle.apply;
+import static org.axonframework.modelling.command.AggregateLifecycle.markDeleted;
+import static org.junit.Assert.*;
+
+public class StubAggregateLifecycleExtensionTest {
```

Comment:
As it's JUnit 5, the `public` modifier can be removed from the class and the methods.